workflows/iuf/operations/managed-nodes/managed-nodes-rollout.yaml

```diff
@@ -189,7 +189,7 @@ spec:
               echo "${prepare_images_output}" | jq ".session_templates" >> "$tmpfile"
               count=$(jq '. | length' "$tmpfile")
               if [[ "$count" == "0" ]]; then
-                  echo "ERROR: No session templates provided to parse, prepare_images_output is empty."
+                  echo "ERROR No session templates provided to parse from 'prepare_images' stage"
```

Comment:
"The managed-nodes-rollout stage relies on the BOS session templates created in the prepare-images stage. Run the prepare-images stage and try again"